test/FsAutoComplete.Tests.Lsp/ExtensionsTests.fs

```diff
@@ -375,3 +375,50 @@ let analyzerTests state =
         do! server.Shutdown()
       })
     ]
+
+let signatureTests state =
+  let server =
+    async {
+      let path = Path.Combine(__SOURCE_DIRECTORY__, "TestCases", "Tooltips")
+      let scriptPath = Path.Combine(path, "Script.fsx")
+      let! (server, events) = serverInitialize path defaultConfigDto state
+      do! waitForWorkspaceFinishedParsing events
+      do! server.TextDocumentDidOpen { TextDocument = loadDocument scriptPath }
+      match! waitForParseResultsForFile "Script.fsx" events with
+      | Ok () ->
+        () // all good, no parsing/checking errors
+      | Core.Result.Error errors ->
+        failtestf "Errors while parsing script %s: %A" scriptPath errors
+
+      return server, scriptPath
+    }
+    |> Async.Cache
+
+  let verifySignature line character expectedSignature =
+    testCaseAsync (sprintf "fsharp/signature for line %d character %d should be '%s'" line character expectedSignature) (async {
+      let! server, scriptPath = server
+      let pos: TextDocumentPositionParams = {
+        TextDocument =  { Uri = sprintf "file://%s" scriptPath }
```

Comment:
There's a utility we use, `Path.FilePathToUri` that we use for these cases to fully-encode the path, because there's some cross-platform nastiness around converting file paths to file URIs. Please use that here as well.